### ISSUE TITLE
Document the expected parameters to twisted.python.log.err

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3134]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3134|
|Type|task|
|Created|2008-03-28 20:06:33Z|

The docstring for the function says:

```
    """
    Write a failure to the log.
    """
```

However, the function takes two named parameters and arbitrary additional keyword arguments.  What do they mean?  What are their expected types?

<details><summary>Searchable metadata</summary>

```
trac-id__3134 3134
type__task task
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1206734793000000 1206734793000000
changetime__1226267743000000 1226267743000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set status to `closed`|
|-|-|

Duplicate of the fixed [#2567](https://github.com/twisted/twisted/issues/2567)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
